.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Build
+
+on:
+    push:
```

Comment:
ah, interesting -- I was unfamiliar with these being used without values to operate on any push, any PR, etc.